docs/moderator.md

```diff
@@ -0,0 +1,86 @@
+> Hello!
+>
+> Welcome to the Angular Community team! We're glad you're here. You're helping
+> ensure the community stays welcoming for everyone. Below, you will find an
+> overview of the server and a frequently asked questions of the most common
+> scenarios you will run into.
```

Comment:
I think you generally talk about `a FAQ`, but not `a frequently asked questions`.

"and some of the most frequently asked questions of common scenarios you might run into."